uui-core/src/data/processing/views/LazyListView.ts

```diff
@@ -278,13 +280,21 @@ export class LazyListView<TItem, TId, TFilter = any> extends BaseListView<TItem,
 
             return { isUpdated, isOutdated, tree: newTree };
 
-        } catch(e) {
+        } catch (e) {
             // TBD - correct error handling
             console.log("LazyListView: Error while loading items");
             return { isUpdated: false, isOutdated: false, tree: loadingTree };
         }
     }
-
+    private collectNotIncludedParents(id: TId, item: TItem): DataRowProps<TItem, TId>[] {
```

Comment:
I believe we need to re-factor the getRowProps. There's a problem:
- we create a tons for DataTableRows
- to put them into getRowProps(_ , _ , parents)
- so the getRowProps gets several of props from the DataRowProps to put into the DataRowProps.path